dags/ods/kktix_ticket_orders/udfs/kktix_api.py

```diff
@@ -25,13 +25,19 @@ def main(**context):
     ts_datetime_obj = parse(context["ts"])
     year = ts_datetime_obj.year
     timestamp = ts_datetime_obj.timestamp()
-    event_raw_data_array = _extract(year=year, timestamp=timestamp,)
+    event_raw_data_array = _extract(
+        year=year,
+        timestamp=timestamp,
+    )
     # load name and email to mailer before data has been hashed
     klaviyo_loader.load(event_raw_data_array)
     transformed_event_raw_data_array = kktix_transformer.transform(event_raw_data_array)
     kktix_loader.load(transformed_event_raw_data_array)
     print(f"Loaded {len(transformed_event_raw_data_array)} rows to BigQuery!")
 
+    # pass these unhashed data through xcom to next airflow task
+    return kktix_transformer._extract_sensitive_unhashed_raw_data(event_raw_data_array)
```

Comment:
simply use a `return` would pass a payload to next airflow task via `XCOM`

this is how `ADD_USER_TO_GATHER_TOWN_WHITELIST` get those sensitive data